src/styles/_variables.scss

```
@@ -0,0 +1,14 @@
+$cell-size: 100px;
+$cell-background-color: #eee;
+$cell-border: 1px solid black;
+$cell-gap: 1px;
+$calendar-width: $cell-size * 7 + 10px;
+
+$start-day:
+  "mon" 0,
+  "tue" $cell-size + $cell-gap,
+  "wed" 2 * $cell-size + $cell-gap,
```

You can calc it just one time instead calc this each time
```suggestion
$calendar-width: $cell-size * 7 + 10px;
$day-margin: $cell-size + $cell-gap;

$start-day:
  "mon" 0,
  "tue" $day-margin,
  "wed" 2 * $day-margin,
```